test/logger.test.js

```diff
@@ -194,4 +195,39 @@ describe('Logger', () => {
 
 	});
 
+	context('ERROR INSTANCE', () => {
+
+		it('returns logger response with masked sensitive data', () => {
+			const data = {
+				responseData: {
+					errorMessage: 'Password field is not defined'
+				},
+				requestData: {
+					rememberMe: true,
+					email: 'email'
+				}
+			};
+			const error = new AnotherError('Something went wrong', data);
+			const loggerResponse = logger.error('Missing fields', error);
+			loggerResponse.should.eql([
+				'Missing fields',
+				{
+					error_data: {
+						requestData: {
+							rememberMe: true,
+							email: '*****'
+						},
+						responseData: {
+							errorMessage: '*****'
```

Comment:
Having this all stared out is not very useful and defeats the point of logging this data. Do we need to improve the masking?

Reply:
I totally agree! Yes, we do. That is its current functionality.

Reply:
This functionality is also resulting in this `info:  source=*****, spoorId=9ad78822-138a-499a-b93f-d4c5547d1bd2, message=init` in next-subscribe. Not ideal.

Reply:
Are you going to address this?

Reply:
I will but in a different PR. I honestly don't know where to start

Reply:
Is this valuable on it's own then? Will it give us any more info or will it all be stared out?

Reply:
So the only reason it is stared out is that it contains the words password and email. Not all responses will have that message. I could change the message in the test but I kind of wanted to make it know that this is an issue.

Reply:
👍